axelor-production/src/main/java/com/axelor/apps/production/service/operationorder/OperationOrderServiceImpl.java

```diff
@@ -168,14 +168,18 @@ public String computeName(ManufOrder manufOrder, int priority, String operationN
   }
 
   @Override
-  public void createToConsumeProdProductList(OperationOrder operationOrder) {
+  public void createToConsumeProdProductList(OperationOrder operationOrder) throws AxelorException {
 
     BigDecimal manufOrderQty = operationOrder.getManufOrder().getQty();
     BigDecimal bomQty = operationOrder.getManufOrder().getBillOfMaterial().getQty();
     ProdProcessLine prodProcessLine = operationOrder.getProdProcessLine();
 
-    if (prodProcessLine.getToConsumeProdProductList() != null) {
+    if (prodProcessLine == null) {
 
+      throw new AxelorException(
+          TraceBackRepository.CATEGORY_NO_VALUE, I18n.get("Your production process is empty."));
```

Comment:
Please define the error message in a constant in [ProductionExceptionMessage.java](https://github.com/axelor/axelor-open-suite/blob/master/axelor-production/src/main/java/com/axelor/apps/production/exceptions/ProductionExceptionMessage.java). Then, please also provide the translations as requested in the ticket.